README.md

```diff
@@ -1,20 +1,49 @@
-# elrrdatasync
+# elrrdatasync API
 ELRR services which aid in the streaming (Kafka streams) of data from the staging database to the ELRR database.
 
-# Requirements
-For building and running the elrrdatasync you need:
-- JDK 1.8
+Setup elrrdatasync first.
+
+# Dependencies
+- Java JDK 17
+- git
 - Maven 3
-# Build the application
-- mvn clean install -Dmaven.test.skip=false
-# Deploying the application on Docker 
-The easiest way to deploy the sample application to Docker is to follow below steps:
-- mkdir -p target/dependency && (cd target/dependency; jar -xf ../*.jar)
-- docker build --build-arg JAR_FILE="./target/elrrdatasync-0.0.1-SNAPSHOT.jar" --file Dockerfile -t <docker_hub>/test:elrrdatasync-dck-img .
-- docker run -p Port:Port -t <docker_hub>/test:elrrdatasync-dck-img
-# Running the application locally
-There are several ways to run a Spring Boot application on your local machine. One way is to execute the main method in the com.deloitte.elrr.datasync.DatasyncApplication class from your IDE
-# Alternatively you can use the Spring Boot Maven plugin: 
-- mvn spring-boot:run
-# Optional step 
-- docker push <docker_hub>/test:elrrdatasync-dck-img
+- Docker Desktop
+- DBeaver
```

Comment:
i mentioned this but DBeaver is just a specific one you/I used, you should just generically refer to it as SQL client throughout, because a user might be using something else. It is not a project dependency.